Frontend/src/app/components/pagination-controls/pagination-controls.component.html

```diff
@@ -1,20 +1,25 @@
-<p>current page: {{ page() }}</p>
-<button
-  (click)="onPageChange(page() - 1)"
-  [disabled]="page() == 1 || loadingService.isLoadingId(loadingId())"
->
-  Prev page
-</button>
-<button
-  (click)="onPageChange(page() + 1)"
-  [disabled]="
-    page() == paginatedResult()?.pagination?.totalPages! ||
-    loadingService.isLoadingId(loadingId()) ||
-    !paginatedResult()?.items?.length
-  "
->
-  Next page
-</button>
+<div class="button-container">
+  <button
+    class="btn"
+    (click)="onPageChange(page() - 1)"
+    [disabled]="page() == 1 || loadingService.isLoadingId(loadingId())"
+  >
+    Prev
+    <ng-icon name="matArrowBack" />
```

Comment:
Would look more normal to have the ng-icon to the left of (or in this case above) "Prev"